run/events-storage/anthos.md

```diff
@@ -0,0 +1,96 @@
+# Events for Cloud Run on Anthos – GCS tutorial
+
+This sample shows how to create a service that processes GCS events.
+
+## Setup
+
+Login to gcloud:
+
+```sh
+gcloud auth login
+```
+
+Configure project id:
+
+```sh
+gcloud config set project [PROJECT-ID]
+```
+
+Configure environment variables:
+
+```sh
+MY_RUN_SERVICE=gcs-service
+MY_RUN_CONTAINER=gcs-container
+MY_GCS_TRIGGER=gcs-trigger
+MY_GCS_BUCKET=gcs-bucket-$(gcloud config get-value project)
+```
+
+## Quickstart
+
+Set cluster name, location and platform:
+
+```sh
+gcloud config set run/cluster events-cluster
```

Comment:
Later I get an error saying: `No cluster named 'events-cluster' in {project_name}`. Can you add a step for creating the clusters?  https://cloud.google.com/run/docs/gke/setup#create_cluster

Reply:
In Pub/Sub tutorial, I say at the beginning "It is assumed that you already created a GKE cluster with Events for Cloud Run already installed." but it looks like I forgot to add a similar note to GCS tutorial. I'll do that tomorrow.

I deliberately left out instructions on creating a GKE cluster with Cloud Run Events because the steps are extremely long and error-prone right now (see steps 6, 7 and 8 of this [codelab](https://codelabs.developers.google.com/codelabs/cloud-run-events-anthos) for details) and they'll get much easier very soon.

Reply:
Can you include the link to the docs for both notes?

Reply:
The docs are not ready yet. There's Alpha User Guide but they'll be somewhat irrelevant soon. I can link to that, if you think it'll be useful.

Reply:
I followed this: https://cloud.google.com/run/docs/gke/setup#create_cluster

Reply:
That creates a GKE cluster with Cloud Run but it doesn't install eventing..

Reply:
I added a note to GCS tutorial that a GKE cluster with Events for Cloud Run is needed for the sample. Again, I feel like we shouldn't link to Alpha User Guide and wait for official docs for installation instructions to link to. Feel free to open a tracking bug and assign to me, if needed.